configurations/pfrich_only.yml

```diff
@@ -1,4 +1,4 @@
 features:
-- beampipe
-- pid:
-    - pfrich
+  beampipe:
```

Comment:
```suggestion
  # beampipe:
```
Prefer to not have the beampipe by default, to be consistent with the dRICH. We use these `_only.yml` configurations for standalone RICH simulation tests.

Reply:
Ah nevermind, I have that in another open PR, no need to do it here

Reply:
Sounds good

> Ah nevermind, I have that in another open PR, no need to do it here